actionmailer/lib/action_mailer/message_delivery.rb

```diff
@@ -59,6 +59,7 @@ def processed?
     # * <tt>:wait</tt> - Enqueue the email to be delivered with a delay
     # * <tt>:wait_until</tt> - Enqueue the email to be delivered at (after) a specific date / time
     # * <tt>:queue</tt> - Enqueue the email on the specified queue
+    # * <tt>:priority</tt> - Enqueues the email with the specified priority
```

Comment:
Lets add an example as well for this above:
`Notifier.welcome(User.first).deliver_later!(priority: 10)`

Reply:
That is a great addition. Added :slightly_smiling_face: